pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -152,6 +152,75 @@ func TestHTTPStats(t *testing.T) {
 	assert.Nil(t, httpReqStats.Stats(500), "500s")            // 500
 }
 
+const (
+	httpbinTestURL = "http://httpbin.org/get"
```

Comment:
not sure if relying at an external service is a good idea. there is a good risk this will be flaky

Reply:
that's the only use-case which reproduced the issue, and we get fin|ack

Reply:
yeah, but you're relying on too many assumptions there:
1) that this service over which we have no control whatsoever will be always available and won't block our CI
2) that it will trigger the scenario you'll trying to add a regression for. but that's zero guarantee that this will be always the case

I _think_ we could potentially try deterministically reproduce this without relying on external servers by setting the socket option `TCP_QUICKACK` and forcing a delay on the server before responding. so the ACK packet would be sent _before_ the response

Reply:
agreed to postpone the test to a different task
https://datadoghq.atlassian.net/browse/USMO-168